From fb364e0b29ce388574a61ebc4a428c559c411a2a Mon Sep 17 00:00:00 2001
From: Peter Dunkley <peter.dunkley@crocodile-rcs.com>
Date: Sun, 5 Jan 2014 18:15:07 +0000
Subject: [PATCH] modules/auth_ephemeral: added OpenSSL licensing exception

(cherry picked from commit 99960be1d55df2ba4ca596c42c8e78e36dad3cf1)
---
 modules/auth_ephemeral/autheph_mod.c | 5 +++++
 modules/auth_ephemeral/autheph_mod.h | 5 +++++
 modules/auth_ephemeral/authorize.c   | 5 +++++
 modules/auth_ephemeral/authorize.h   | 5 +++++
 modules/auth_ephemeral/checks.c      | 5 +++++
 modules/auth_ephemeral/checks.h      | 5 +++++
 6 files changed, 30 insertions(+)

diff --git a/modules/auth_ephemeral/autheph_mod.c b/modules/auth_ephemeral/autheph_mod.c
index 8e72fba..fb59e56 100644
--- a/modules/auth_ephemeral/autheph_mod.c
+++ b/modules/auth_ephemeral/autheph_mod.c
@@ -19,6 +19,11 @@
  * along with this program; if not, write to the Free Software 
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
+ * Exception: permission to copy, modify, propagate, and distribute a work
+ * formed by combining OpenSSL toolkit software and the code in this file,
+ * such as linking with software components and libraries released under
+ * OpenSSL project license.
+ *
  */
 #include "../../dprint.h"
 #include "../../locking.h"
diff --git a/modules/auth_ephemeral/autheph_mod.h b/modules/auth_ephemeral/autheph_mod.h
index 62bc808..3ad2356 100644
--- a/modules/auth_ephemeral/autheph_mod.h
+++ b/modules/auth_ephemeral/autheph_mod.h
@@ -19,6 +19,11 @@
  * along with this program; if not, write to the Free Software 
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
+ * Exception: permission to copy, modify, propagate, and distribute a work
+ * formed by combining OpenSSL toolkit software and the code in this file,
+ * such as linking with software components and libraries released under
+ * OpenSSL project license.
+ *
  */
 #ifndef AUTHEPH_MOD_H
 #define AUTHEPH_MOD_H
diff --git a/modules/auth_ephemeral/authorize.c b/modules/auth_ephemeral/authorize.c
index a09a9f8..454ecac 100644
--- a/modules/auth_ephemeral/authorize.c
+++ b/modules/auth_ephemeral/authorize.c
@@ -19,6 +19,11 @@
  * along with this program; if not, write to the Free Software 
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
+ * Exception: permission to copy, modify, propagate, and distribute a work
+ * formed by combining OpenSSL toolkit software and the code in this file,
+ * such as linking with software components and libraries released under
+ * OpenSSL project license.
+ *
  */
 #include <openssl/hmac.h>
 #include <openssl/sha.h>
diff --git a/modules/auth_ephemeral/authorize.h b/modules/auth_ephemeral/authorize.h
index 8ce9331..0a7bb91 100644
--- a/modules/auth_ephemeral/authorize.h
+++ b/modules/auth_ephemeral/authorize.h
@@ -19,6 +19,11 @@
  * along with this program; if not, write to the Free Software 
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
+ * Exception: permission to copy, modify, propagate, and distribute a work
+ * formed by combining OpenSSL toolkit software and the code in this file,
+ * such as linking with software components and libraries released under
+ * OpenSSL project license.
+ *
  */
 #ifndef AUTHORIZE_H
 #define AUTHORIZE_H
diff --git a/modules/auth_ephemeral/checks.c b/modules/auth_ephemeral/checks.c
index 447e997..dacec11 100644
--- a/modules/auth_ephemeral/checks.c
+++ b/modules/auth_ephemeral/checks.c
@@ -19,6 +19,11 @@
  * along with this program; if not, write to the Free Software 
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
+ * Exception: permission to copy, modify, propagate, and distribute a work
+ * formed by combining OpenSSL toolkit software and the code in this file,
+ * such as linking with software components and libraries released under
+ * OpenSSL project license.
+ *
  */
 #include "../../dprint.h"
 #include "../../mod_fix.h"
diff --git a/modules/auth_ephemeral/checks.h b/modules/auth_ephemeral/checks.h
index 27cb7fe..ee23c7e 100644
--- a/modules/auth_ephemeral/checks.h
+++ b/modules/auth_ephemeral/checks.h
@@ -19,6 +19,11 @@
  * along with this program; if not, write to the Free Software 
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
+ * Exception: permission to copy, modify, propagate, and distribute a work
+ * formed by combining OpenSSL toolkit software and the code in this file,
+ * such as linking with software components and libraries released under
+ * OpenSSL project license.
+ *
  */
 #ifndef CHECKS_H
 #define CHECKS_H
-- 
1.8.5.2

